runtime/development/Cargo.toml

```diff
@@ -115,6 +115,7 @@ pallet-claims = { path = "../../pallets/claims", default-features = false }
 pallet-collator-allowlist = { path = "../../pallets/collator-allowlist", default-features = false }
 pallet-connectors = { path = "../../pallets/connectors", default-features = false }
 pallet-connectors-gateway = { path = "../../pallets/connectors-gateway", default-features = false }
+gateway-routers = { path = "../../pallets/connectors-gateway/gateway-routers", default-features = false }
```

Comment:
I feel a bit uncomfortable that these sub pallets of connectors are not named accordingly 🤔 

I would even suggest that we do:

```
- pallets/
  - connectors/                    <--- "pallet-connectors"
  - connectors-gateway/    <--- "pallet-connectors-gateway"
    - routers                          <--- "pallet-connectors-gateway-routers"
```

Or even have `connectors-gateway/` under `connectors/` and just name it `gateway` while still keeping the absolute name `pallet-connectors-gateway`. I don't feel strongly about this last suggestion but we should at least consider the first one.